.dependency-cruiser.js

```diff
@@ -276,7 +276,7 @@ module.exports = {
        defaults to './tsconfig.json'.
      */
     tsConfig: {
-      fileName: 'tsconfig.json',
+      fileName: 'tsconfig.production.json',
```

Comment:
This change is unneeded. Easier to rename `tsconfig.production.json` to `tsconfig.json`